courses/free-courses-ar.md

```diff
@@ -257,6 +238,34 @@ Emam Academy
 * [Vue.js Router Tutorial](https://www.youtube.com/playlist?list=PLfDx4cQoUNObtw2FtE4_eX_k9yCf-Fcd3) - Algorithm Academy
 
 
+### Logic Programming - Prolog
+
+* [Logic Programming - Prolog - برمجة منطقية](https://www.youtube.com/playlist?list=PLMm8EjqH1EFW9Faldu6D6Uh2j1EWWaTYe) - Ahmed Sallam
+
+
+### Object Oriented Paradigm
+
+* [C++ Object-Oriented Design and Programming](https://www.youtube.com/playlist?list=PLPt2dINI2MIbMba7tpx3qvmgOsDlpITwG) -  Arabic Competitive Programming
```

Comment:
Better in C++ category

Reply:
Yes, it's best in C++, so when I noticed it was there I erased it.